### ISSUE TITLE
Indoor : ajouter le bouton pour garder un commentaire privé

### ISSUE THREAD

**Issue by StephaneMichelet** [oblyk/oblyk-app#46]
Sur les voies extérieures quand on fait une voie, on peut décider de garder le commentaire privé.
Mais en indoor, il n'y a pas la petite croix pour cela.

**Comment by lucien-chastan** [comment]
Salut !
Je me demandais si plutôt que faire un commentaire privé ou public, je ne ferais pas deux champs, une note pour soit et un commentaire pour les autres ...
Je me suis déjà retrouvé dans la situation ou je voulais faire les deux

**Comment by StephaneMichelet** [comment]
Effectivement, je me suis fais la même remarque : je ne fais quasi que des commentaires en privé, car c'est pour moi. Mais parfois, j'en fais un public, sauf que c'est censé être rattaché à une tentative.

Donc un "avis" pourrait être un commentaire public, car c'est utile pour les autres !
